server/activate_hooks.go

```diff
@@ -96,15 +94,5 @@ func (p *Plugin) OnDeactivate() error {
 }
 
 func (p *Plugin) checkRequiredServerConfiguration() (bool, error) {
-	path, err := p.API.GetBundlePath()
-	if err != nil {
-		return false, errors.Wrap(err, "failed to get bundle path")
-	}
-
-	manifest, _, err := model.FindManifest(path)
-	if err != nil {
-		return false, errors.Wrap(err, "failed to find manifest file")
-	}
-
```

Comment:
Is this intentionally deleted?

Reply:
Yes. The method can reuse https://github.com/mattermost/mattermost-plugin-demo/pull/91/files#diff-3407a86002fad9e52eb18c8919aa27b5R11 and doesn't have to fetch the manifest itself